activerecord/lib/active_record/transactions.rb

```diff
@@ -267,6 +267,14 @@ def after_rollback(*args, &block)
       end
 
       private
+        def prepend_option
+          if ActiveRecord.run_after_transaction_callbacks_in_order_defined
+            { prepend: true }
+          else
+            {}
+          end
+        end
+
```

Comment:
Would this prevent someone setting `prepend: false` on an individual callback to get the old behavior?

Reply:
Hmmm, I guess so.

I don't think `prepend: false` does anything at the moment. I'm not sure it would make sense (except in the context of this change...).

Reply:
But I guess maybe we should support `prepend: true` somehow 🤔

Reply:
Geez. What a rabbit hole. https://github.com/rails/rails/issues/47007

Reply:
Based on the response to #47007 (@rafaelfranca confirmed if it is working as expected), I don't think we should add any special behaviour for `prepend` here. That way this PR will make `after_commit` be consistent with all other `after_` callbacks.